server/src/controllers/v1/cards.js

```diff
@@ -10,7 +10,13 @@ module.exports = {
       .then((card) => { res.status(201).send(card) })
       .catch((error) => { res.status(400).send(error) })
   },
-
+  destroy: (req, res) => {
+    Card
+      .findByPk(req.params.id)
+      .then((card) => { card.destroy() })
+      .then(() => { res.status(204).send() })
+      .catch(() => { res.status(400).send({ error: 'Bad Request', message: 'Card not found' }) })
+  },
   getAll: (req, res) => {
```

Comment:
@nikitasakov can we leave empty line between actions? So the code can be more readable